### PR TITLE
feat(tornado_client.py): support tornado http client calls

### PR DESCRIPTION
**Pull request by ranrib**

### CODE REVIEW

File: epsagon/modules/tornado.py [hunk 1]
```diff
@@ -229,6 +235,58 @@ def thread_pool_execution(cls, unique_id, fn, args, kwargs):
...
+    :param kwargs: wrapt's kwargs
+    :return: None
+    """
+    try:
+        request, raise_error = _prepare_http_request(*args, **kwargs)
+    except Exception:  # pylint: disable=W0703
+        return wrapped(*args, **kwargs)
+
+    # Inject header to support tracing over HTTP requests
+    trace_id = uuid.uuid4().hex
```
Comment (razlo) [inline epsagon/modules/tornado.py:265]: I'd maybe export this header creation logic somewhere since we do it everywhere now

**Review by razlo — COMMENTED** [review]
looks good overall
can you please add some UT? (even a sanity test)

File: epsagon/modules/tornado.py (hunk 1 above)
Comment (ranrib) [reply epsagon/modules/tornado.py]: Done

**Comment by ranrib** [comment]
Added test @razlo

**Review by razlo — DISMISSED** [review]

**Review by razlo — APPROVED** [review]

**Pull request merged**